Tests/Tests/Saml2P/Saml2ResponseTests.cs

```diff
@@ -1778,6 +1779,7 @@ public void Saml2Response_Xml_FromData_ContainsAudienceRestriction()
                 new Saml2Id("InResponseToID"),
                 null,
                 new Uri(audience),
+                false,
```

Comment:
Revert this change. Having to change this is a sign of changing the public API (as commented previously)